src/main/java/javax/validation/Configuration.java

```diff
@@ -234,6 +249,20 @@
 	ParameterNameProvider getDefaultParameterNameProvider();
 
 	/**
+	 * Returns an implementation of the {@link ClockProvider}
+	 * interface following the default {@code ClockProvider}
+	 * defined in the specification:
+	 *
+	 * XXX BVAL-496 paste definition of the specification
```

Comment:
Do you mind if i change that into "TODO". Seems a bit more common. Also this one is the only occurence I could find.